CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 *	Architecture detection using uname instead of dpkg [\#3948](https://github.com/rvm/rvm/issues/3948)
 * Help section of the rvm.io page points to irc.freenode.net instead of github [\#3939](https://github.com/rvm/rvm/issues/3939)
 * Make sure stderr output is printed on console and not captured into log files (No ticket)
+* Failing openssl.patch for Ruby 2.2.4 and 2.2.5 [\#3988](https://github.com/rvm/rvm/issues/3988)
```

Comment:
I mean - we should use language that describes the action, right now the message does not say anything about the action, someone can assume we are adding failing `openssl.patch` - and wonder `why someone would add a failing patch?` :)